torch/distributions/bernoulli.py

```diff
@@ -25,8 +26,16 @@ class Bernoulli(Distribution):
     """
     has_enumerate_support = True
 
-    def __init__(self, probs):
-        self.probs, = broadcast_all(probs)
+    def __init__(self, probs=None, logits=None):
+        if (probs is None) == (logits is None):
+            raise ValueError("Got probs={}, logits={}. Either `probs` or `logits` must be specified, "
+                             "but not both.".format(probs, logits))
```

Comment:
I'm afraid that this will print a huge message containing two tensors. I think it's enough to just say that exactly one of them should be specified.

Reply:
Done.